docs/helm.md

```diff
@@ -125,6 +125,7 @@ HELM_VALUES=('shared/app,staging/app')
 In some cases, it may make more sense to use a remote chart from a Helm repository. This can be accomplished with a couple extra parameters.
 
 ```bash
+REMOTE_CHART=true
```

Comment:
Isn't it always a remote chart if we specify HELM_REPO_URLS?  Maybe we can just use that.

Reply:
👍